imports/imports.go

```diff
@@ -117,15 +137,58 @@ func str2RE(vs []string) ([]*regexp.Regexp, error) {
 	return o, nil
 }
 
-// Error stores information about a deprecated import in a Go file.
+func includeStd(list []string) ([]string, error) {
+	if len(list) == 0 {
+		return list, nil
+	}
+	pkgs, err := std()
+	if err != nil {
+		return nil, err
+	}
+	return append(list, pkgs...), nil
+}
+
+// str returns a slice of patterns matching the standard library.
+func std() ([]string, error) {
```

Comment:
alternatively, this could be pre-generated with a `//go:generate go list std` directive.
(one for each Go version we support, "of course.")

Reply:
I think this is a lower maintenance burden.

Reply:
ok.